<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" ignored="0" total="14" passed="11">
  <reporter-output>
  </reporter-output>
  <suite name="Command line suite" duration-ms="52840" started-at="2018-08-03T08:30:22Z" finished-at="2018-08-03T08:31:15Z">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="52840" started-at="2018-08-03T08:30:22Z" finished-at="2018-08-03T08:31:15Z">
      <class name="com.icreon.account.firstLoginTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.icreon.account.firstLoginTest@2c6a3f77]" name="setUp" is-config="true" duration-ms="355" started-at="2018-08-03T08:30:22Z" finished-at="2018-08-03T08:30:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="loginPortal()[pri:0, instance:com.icreon.account.firstLoginTest@2c6a3f77]" name="loginPortal" duration-ms="6489" started-at="2018-08-03T08:30:22Z" finished-at="2018-08-03T08:30:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPortal -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.icreon.account.firstLoginTest@2c6a3f77]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-08-03T08:30:29Z" finished-at="2018-08-03T08:30:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.icreon.account.firstLoginTest -->
      <class name="com.icreon.customer.kidTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.icreon.customer.kidTest@62043840]" name="setUp" is-config="true" duration-ms="0" started-at="2018-08-03T08:30:36Z" finished-at="2018-08-03T08:30:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="TC_001_SavingANewKid()[pri:0, instance:com.icreon.customer.kidTest@62043840]" name="TC_001_SavingANewKid" duration-ms="1337" started-at="2018-08-03T08:30:36Z" finished-at="2018-08-03T08:30:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_001_SavingANewKid -->
      </class> <!-- com.icreon.customer.kidTest -->
      <class name="com.icreon.customer.MileagePlusRewardNumberTest">
        <test-method status="FAIL" signature="TC_001_MileageNumberAlreadyExist()[pri:0, instance:com.icreon.customer.MileagePlusRewardNumberTest@5315b42e]" name="TC_001_MileageNumberAlreadyExist" duration-ms="31137" started-at="2018-08-03T08:30:37Z" finished-at="2018-08-03T08:31:08Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[1 expectation failed.
Expected status code <200> but was <404>.
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <200> but was <404>.

at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:83)
at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:77)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:60)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:235)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:247)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:483)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:655)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:210)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:59)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:166)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:123)
at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:131)
at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:119)
at com.icreon.customer.MileagePlusRewardNumberTest.TC_001_MileageNumberAlreadyExist(MileagePlusRewardNumberTest.java:36)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_001_MileageNumberAlreadyExist -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.icreon.customer.MileagePlusRewardNumberTest@5315b42e]" name="setUp" is-config="true" duration-ms="0" started-at="2018-08-03T08:30:37Z" finished-at="2018-08-03T08:30:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="TC_002_MileageNumberNotValid()[pri:0, instance:com.icreon.customer.MileagePlusRewardNumberTest@5315b42e]" name="TC_002_MileageNumberNotValid" duration-ms="573" started-at="2018-08-03T08:31:08Z" finished-at="2018-08-03T08:31:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_002_MileageNumberNotValid -->
      </class> <!-- com.icreon.customer.MileagePlusRewardNumberTest -->
      <class name="com.icreon.customer.customerProfileInformationTest">
        <test-method status="PASS" signature="TC_001_Passing0InPath()[pri:0, instance:com.icreon.customer.customerProfileInformationTest@246ae04d]" name="TC_001_Passing0InPath" duration-ms="2548" started-at="2018-08-03T08:30:29Z" finished-at="2018-08-03T08:30:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_001_Passing0InPath -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.icreon.customer.customerProfileInformationTest@246ae04d]" name="setUp" is-config="true" duration-ms="0" started-at="2018-08-03T08:30:29Z" finished-at="2018-08-03T08:30:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="TC_002_PassingProfileIDinPath()[pri:0, instance:com.icreon.customer.customerProfileInformationTest@246ae04d]" name="TC_002_PassingProfileIDinPath" duration-ms="517" started-at="2018-08-03T08:30:31Z" finished-at="2018-08-03T08:30:32Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [404] but found [200]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [404] but found [200]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.icreon.customer.customerProfileInformationTest.TC_002_PassingProfileIDinPath(customerProfileInformationTest.java:91)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_002_PassingProfileIDinPath -->
        <test-method status="PASS" signature="TC_003_PassingWrongProfileIDinPath()[pri:0, instance:com.icreon.customer.customerProfileInformationTest@246ae04d]" name="TC_003_PassingWrongProfileIDinPath" duration-ms="581" started-at="2018-08-03T08:30:32Z" finished-at="2018-08-03T08:30:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_003_PassingWrongProfileIDinPath -->
        <test-method status="PASS" signature="TC_004_requestWithoutAuthorization()[pri:0, instance:com.icreon.customer.customerProfileInformationTest@246ae04d]" name="TC_004_requestWithoutAuthorization" duration-ms="963" started-at="2018-08-03T08:30:33Z" finished-at="2018-08-03T08:30:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_004_requestWithoutAuthorization -->
        <test-method status="PASS" signature="TC_005_requestWithoutX_Fred_Auth()[pri:0, instance:com.icreon.customer.customerProfileInformationTest@246ae04d]" name="TC_005_requestWithoutX_Fred_Auth" duration-ms="571" started-at="2018-08-03T08:30:33Z" finished-at="2018-08-03T08:30:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_005_requestWithoutX_Fred_Auth -->
        <test-method status="PASS" signature="TC_006_requestWithoutCustomerId()[pri:0, instance:com.icreon.customer.customerProfileInformationTest@246ae04d]" name="TC_006_requestWithoutCustomerId" duration-ms="484" started-at="2018-08-03T08:30:34Z" finished-at="2018-08-03T08:30:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_006_requestWithoutCustomerId -->
        <test-method status="PASS" signature="TC_007_mismatchedCustomerId()[pri:0, instance:com.icreon.customer.customerProfileInformationTest@246ae04d]" name="TC_007_mismatchedCustomerId" duration-ms="487" started-at="2018-08-03T08:30:35Z" finished-at="2018-08-03T08:30:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_007_mismatchedCustomerId -->
        <test-method status="PASS" signature="custProfileInfo()[pri:0, instance:com.icreon.customer.customerProfileInformationTest@246ae04d]" name="custProfileInfo" duration-ms="734" started-at="2018-08-03T08:30:35Z" finished-at="2018-08-03T08:30:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- custProfileInfo -->
      </class> <!-- com.icreon.customer.customerProfileInformationTest -->
      <class name="com.icreon.customer.stravaUpdateTest">
        <test-method status="PASS" signature="TC_001_PassingAllParametersCorrectly()[pri:0, instance:com.icreon.customer.stravaUpdateTest@2ef9b8bc]" name="TC_001_PassingAllParametersCorrectly" duration-ms="2008" started-at="2018-08-03T08:31:09Z" finished-at="2018-08-03T08:31:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_001_PassingAllParametersCorrectly -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.icreon.customer.stravaUpdateTest@2ef9b8bc]" name="setUp" is-config="true" duration-ms="0" started-at="2018-08-03T08:31:09Z" finished-at="2018-08-03T08:31:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="TC_002_UsingExsitingStravaId()[pri:0, instance:com.icreon.customer.stravaUpdateTest@2ef9b8bc]" name="TC_002_UsingExsitingStravaId" duration-ms="4010" started-at="2018-08-03T08:31:11Z" finished-at="2018-08-03T08:31:15Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Success] but found [Starva Id already exist]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Success] but found [Starva Id already exist]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.icreon.customer.stravaUpdateTest.TC_002_UsingExsitingStravaId(stravaUpdateTest.java:80)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_002_UsingExsitingStravaId -->
      </class> <!-- com.icreon.customer.stravaUpdateTest -->
    </test> <!-- Command line test -->
  </suite> <!-- Command line suite -->
</testng-results>
